res/layout/host_information.xml

```diff
@@ -220,7 +265,8 @@
                             android:layout_width="wrap_content"
                             android:layout_height="wrap_content"
                             android:layout_marginLeft="56dp"
-                            android:text="I&apos;m the Webmaster of Warmshowers.org and now the Executive Director as well, the first paid staff for our great organization. " />
+                            android:text="I&apos;m the Webmaster of Warmshowers.org and now the Executive Director as well, the first paid staff for our great organization. "
```

Comment:
I guess we should be using default text that is simpler than this. This must have come from when I was mocking it up. No need to respond :-)